### DISCUSSION TITLE
[x][postponed][privacy][trackhub registry][trackfind] For the study->orcid attribute, we have a comment "ORCID - Needs GDPR consent".

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#116] [Team Posts]
What were the thoughts about this?

**Comment by dzerbino** [comment]
This would require an amendment to the TrackHub Registry data protection statement. It's fairly straightforward, 

It's not required at the very moment, since the ORCID on the Blueprint trackhub is mine, and I'm not going to sue myself ;)

**Comment by sveinugu** [comment]
@dtitov We would then need to add similar info to TrackFind also, I suppose

**Comment by sveinugu** [comment]
Linked to in https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/49. Removing pin
